### DISCUSSION TITLE
FAQ: Add the meaning of "Disabled by Owner"

### DISCUSSION THREAD

**Discussion by ackvf** [Anarios/return-youtube-dislike#987] [Q&A]
I recently came across a dislike button that says "Disabled by Owner".  was really confused, because there was no information about this on extension market page or on the website or FAQ. Please add it to FAQ - why this appears and how, as an owner, can somebody apply for this.

**Comment by Anarios** [comment]
It's a YouTube setting to hide all ratings. Without rating data there's no way to produce meaningful prediction of the score

**Reply by ackvf** [reply]
I see, thank you for the explanation. Could you perhaps add it to to FAQ and a small note directly on the hover of the button?

**Reply by munael** [reply]
Seconding the call to add it as a tooltip on the button.

**Reply by maujogador** [reply]
Another year, another request
